bouncer/util.py

```diff
@@ -44,15 +45,31 @@ def parse_document(document):
     annotation_id = document["_id"]
     annotation = document["_source"]
 
+    # And that annotations always have "group" and "shared"
+    group = annotation["group"]
+    shared = annotation["shared"]
```

Comment:
I like that better too. Although pep8 complains, and seems to prefer `is` to `==`,  I can't totally follow the arguments pro and con, so I'll use `is` for now and defer to your judgement.